app/uk/gov/hmrc/pdfgenerator/controllers/HtmlSupport.scala

```diff
@@ -26,12 +27,12 @@ trait HtmlSupport {
 
   val html: Mapping[String] = text verifying (Constraints.nonEmpty , noScriptTags)
 
-  def createForm() = {
-    Form(
-      single(
-        "html" -> html
-      )
-    )
-  }
+  case class PdfForm(html: String, createPdfA: Boolean)
 
+  def getPdfForm() = Form(
+      mapping(
+        "html" -> html,
+        "create-pdfa" -> default(boolean, false)
```

Comment:
The current behaviour is that it will always create PDF/A but PDF/A does not allow hyperlinks. We therefore implemented the regex link-checking to bypass PDF/A generation _if_ the document contains valid gov.uk links and no other kinds of links. This means that any document _without_ links will still be PDF/A. This boolean parameter overrides the link-based selection to force PDF/A generation. Until calling services are updated to use this parameter it will have no way of generating PDFs with links - which is why this was set to false.